app/controllers/api/v1x0/graphql_controller.rb

```diff
@@ -8,7 +8,7 @@ class GraphqlController < ApplicationController
       def overlay
         {
           "^.*$" => {
-            "base_query" => lambda do |model_class, _ctx|
+            "base_query" => lambda do |model_class, _args, _ctx|
```

Comment:
Should this change be made only in the v1x1 version of graphql_controller?

Reply:
@syncrou Since this is internal and we can't keep the old code and the new code in a single app when we can only have a single insights-api-common gem. Its not possible to carry 2 insights-api-common in a single app.

I don't think we can do this in a Gemfile
```
gem 'insights-api-common', '~> 3.8'
gem 'insights-api-common', '~> 4.0'
```

Reply:
Makes sense.